### ISSUE TITLE
numpy<1.20.0 required

### ISSUE THREAD

**Issue by KoichiYasuoka** [adobe/NLP-Cube#136]
`transformers==4.2.2` seems too old and requires `numpy<1.20.0`. Please re-consider their versions in [requirements.txt](https://github.com/adobe/NLP-Cube/blob/master/requirements.txt).

**Comment by tiberiu44** [comment]
Hi again @KoichiYasuoka , and thank you for noticing this. We will update the requirement file with newer versions for transformers and consequently numpy. However, we do have comaptibility issues with numpy >1.23 and that will need to be addressed. The update will go out at the end of next week, because everyone working on the project is caught with other work.

**Comment by tiberiu44** [comment]
Hi @KoichiYasuoka ,

I've created a PR. Once it is approved, I will also update the pip package and upload.

**Comment by tiberiu44** [comment]
The update is merged into main and released as nlpcube version 3.1.2.
